lib/Alchemy/Phrasea/Status/StatusCollection.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+/*
+ * This file is part of Phraseanet
+ *
+ * (c) 2005-2015 Alchemy
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Alchemy\Phrasea\Status;
+
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\Common\Collections\Collection;
+
+/**
+ * This class represents a collection of status
+ */
+class StatusCollection extends ArrayCollection
+{
```

Comment:
These methods should be on `StatusStructure` object, since they manipulate the structure.